backend/src/main/java/com/crm/api/v1/CarController.java

```diff
@@ -0,0 +1,26 @@
+package com.crm.api.v1;
+
+import com.crm.dto.response.CarResponse;
+import com.crm.service.CarServiceImpl;
+import org.springframework.data.domain.Page;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/cars")
+public class CarController {
+
+    private final CarServiceImpl carService;
+
+    public CarController(CarServiceImpl carService) {
+        this.carService = carService;
+    }
+
+    @GetMapping("")
+    public Page<CarResponse> getCarsPaginated(@RequestParam(required = false, defaultValue = "0") int page,
```

Comment:
You don't need defaultValue = "0" because you are using primitive type.

Reply:
Done